### PR TITLE
feat(components/atom/label): Change the * of the Optional Text to Opt…

### LINKED ISSUE #1752: Label DEMO - Improvements
# Acceptance Criteria
For the demo of [Labels](https://sui-components.vercel.app/workbench/atom/label/demo)

- [ ] Use SUI Inputs, Checkboxes and Buttons
- [ ] Change the * of the Optional Text to "Optional"

labels

### PR DESCRIPTION
**Pull request by stefi23**
…ional

## Atom/label
**TASK**: #1752

### Types of changes

- [x] Documentation Only
- [x] Refactor

### Description, Motivation and Context

- Changes to demo documentation
 - replaced input, radio and button to use the SUI Inputs, Checkboxes and Buttons components
-  Changed the * of the Optional Text to "(Optional)"

### Screenshots - Animations
TBA

### CODE REVIEW

File: components/atom/input/demo/styles.scss [hunk 1]
```diff
@@ -1,3 +1,3 @@
-@import '~@s-ui/react-atom-button/lib/index.scss';
```
Comment (andresin87) [inline components/atom/input/demo/styles.scss:1]: can you fix it on ahother different pr?

**Comment by andresin87** [comment]
https://github.com/SUI-Components/sui-components/pull/1821

**Comment by stefi23** [comment]
The demo didn't work for the sui components I've added(input, checkbox and button), although dependencies where added. I was about to ask. Tried also importing the component styles and it didn t work either

File: components/atom/input/demo/styles.scss (hunk 1 above)
Comment (stefi23) [reply components/atom/input/demo/styles.scss:1]: @andresin87 Yes, you mean to remove this line?

**Pull request closed without merging**